cli/js/os_test.ts

```diff
@@ -116,6 +116,11 @@ test(function osPid(): void {
   assert(Deno.pid > 0);
 });
 
+test(function testGetPriority(): void {
```

Comment:
How about a test of `setPriority()` as well?  You could do something as easy as:
```
const priority = Deno.getPriority();
Deno.setPriority(-20, 0);
assertEquals(Deno.getPriority(), -20);
Deno.setPriority(priority, 0);  //e.g. reset process priority
```

Reply:
i'll add more then.